src/spoke/Spoke.sol

```diff
@@ -853,16 +855,19 @@ abstract contract Spoke is ISpoke, Multicall, NoncesKeyed, AccessManagedUpgradea
       Reserve storage reserve = _reserves[reserveId];
       IHubBase hub = reserve.hub;
       uint256 assetId = reserve.assetId;
+
       uint256 drawnIndex = hub.getAssetDrawnIndex(assetId);
       (uint256 drawnDebtReported, uint256 premiumDebtRay) = userPosition.getDebt(drawnIndex);
+      uint256 deficitShares = drawnDebtReported.rayDivDown(drawnIndex);
```

Comment:
should we abstract away this in a fn? not sure where it would leave, wdyt? overkill?

Reply:
not a fan for such a simple conversion, better keep it like that.